cime_config/machines/config_machines.xml

```diff
@@ -216,7 +216,7 @@
 
       <modules compiler="intel">
         <command name="load">PrgEnv-intel/8.3.3</command>
-        <command name="load">intel/2023.0.0</command>
+        <command name="load">intel/2023.1.0</command>
```

Comment:
Not sure why this is showing as a diff.  Current master has 2023.1.0.

Reply:
The branch was lagging master on this change. It was manually updated in order to use pm-cpu_intel for additional v3alpha02 simulations (#5798).

Reply:
Don't need a rebase.  Just make sure it winds up not changing config_machines.